tests/system/libraries/AssertsLib.py

```diff
@@ -13,6 +13,10 @@
 class AssertsLib:
 	@staticmethod
 	def strings_match(actual, expected, ignore_case=False):
+		builtIn.log(

```

Comment:
Add comment: 'Include text in robot test report so that actual behavior can be determined (using the report) even when the test passes.'